exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/otlp/OtlpUserAgent.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.internal.otlp;
+
+import io.opentelemetry.sdk.resources.Resource;
+import java.util.Properties;
+import java.util.function.BiConsumer;
+
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
+public class OtlpUserAgent {
+
+  private static final String userAgent;
+
+  static {
+    userAgent = "OTel OTLP Exporter Java/" + readVersion();
+  }
+
+  private static String readVersion() {
+    Properties properties = new Properties();
+    try {
+      properties.load(
+          Resource.class.getResourceAsStream(
+              "/io/opentelemetry/exporter/internal/otlp/version.properties"));
```

Comment:
I almost never use relative paths for loading resources, but maybe this is why that functionality exists?
```suggestion
      properties.load(Resource.class.getResourceAsStream("version.properties"));
```

Reply:
Yeah I didn't know that would work. Feels mysterious that this works, but it does and is more readable 👍

Reply:
Yup! I caught that and have it committed locally, just forgot to push it up (hence all the resolved comments without a corresponding commit 😬).